examples/state_space_connectivity.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun 14 13:15:43 2022
+
+@author: jordandrew
+
+For 'mne-connectivity/examples/' to show usage of MEGLDS 
+Use MNE-sample-data for auditory/left
+"""
+
+## import necessary libraries
+import mne
+import numpy as np 
+import matplotlib.pyplot as plt
+from mne_util import ROIToSourceMap, scale_sensor_data #mne_util is from MEGLDS repo
+
+## define paths to sample data
+data_path = '/Users/jordandrew/Documents/MEG/mne_data/MNE-sample-data'
+# data_path = mne.datasets.sample.data_path()
+sample_folder = '/MEG/sample'
+raw_fname = data_path + sample_folder + '/sample_audvis_raw.fif' #how many subjects?
+subjects_dir = data_path + '/subjects'
+
+## import raw data and find events 
+raw = mne.io.read_raw_fif(raw_fname).crop(tmax=60)
+events = mne.find_events(raw, stim_channel='STI 014')
+""" OR
+raw_events_fname = data_path + sample_folder + '/sample_audvis_raw-eve.fif'
+events = mne.read_events(raw_events_fname)
+"""
+
+## compute forward solution
+sphere = mne.make_sphere_model('auto', 'auto', raw.info)
+src = mne.setup_volume_source_space(sphere=sphere, exclude=30., pos=15.)
+fwd = mne.make_forward_solution(raw.info, trans=None, src=src, bem=sphere)
+fwd['src'].append( fwd['src'][0]) #fwd['src'] needs lh and rh; duplicated here
+#is there a reason the sample data only has 1 hemisphere of data?
+
+## define epochs using event_dict
+# event_id = 1
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'face': 5, 'buttonpress': 32}
+epochs = mne.Epochs(raw, events, tmin=-0.3, tmax=0.7, event_id=event_dict,
+                    preload=True)
+# del raw
+
+## compute covariance 
+noise_cov = mne.compute_covariance(epochs, tmax=0)
+labels = mne.read_labels_from_annot('sample', subjects_dir=subjects_dir)
+roi_to_src = ROIToSourceMap(fwd, labels) # compute ROI-to-source map
```

Comment:
To me this is where it should be `model = LDS(fwd, labels, noise_cov, epochs, subjects_dir)`. Or if we're planning on the model being fit over multiple subjects:
```
model = LDS(..., subjects_dir=None)
model.add_subject(subject, fwd, labels, noise_cov, epochs)
model.fit()
```
and at the `add_subject` stage it could do all the whitening/rescaling etc. And at the `.fit` stage it does all the heavy lifting. And the `...` above could be all the stuff about lambdas, `random_state` support, etc. We'd have to see what the minimum set of parameters is we need for the `LDS` class...